src/Illuminate/Routing/composer.json

```diff
@@ -39,7 +39,8 @@
     },
     "suggest": {
         "illuminate/console": "Required to use the make commands (^6.0).",
-        "symfony/psr-http-message-bridge": "Required to psr7 bridging features (^1.2)."
+        "nyholm/psr7": "Required to use PSR-7 bridging features (^1.2).",
```

Comment:
Please also add `laminas/laminas-diactoros` (to both composer.json files).

Reply:
Pausing on this until we agree on how to proceed regarding the current lack of a DiactorosFactory.